### PR TITLE
Fix desktop-related files

### PR DESCRIPTION
**Pull request by eunos-1128**
I fixed desktop-related files which have wrong format for flatpak packaging.

### CODE REVIEW

File: coot.desktop [hunk 1]
```diff
@@ -4,7 +4,7 @@ Name=Coot
 GenericName=Crystallographic Object-Oriented Toolkit
 Comment=Macromolecular model builder
 Exec=coot %f
-Icon=coot
+Icon=io.github.pemsley.coot
 Terminal=true
```
[the remark this answers is not in the document]
Comment (pemsley) [reply coot.desktop]: Yes... it does need a terminal at the moment. I will replace that with internal logging one day.

[the remark this answers is not in the document]
Comment (pemsley) [reply coot.desktop]: The icon should be the one on the web page:
https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/web/coot-with-ATP-vector.png

[the remark this answers is not in the document]
Comment (eunos-1128) [reply coot.desktop]: > The icon should be the one on the web page: [www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/web/coot-with-ATP-vector.png](https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/web/coot-with-ATP-vector.png)

Thanks, I'll process it into SVG to use it as icon.

[the remark this answers is not in the document]
Comment (eunos-1128) [reply coot.desktop]: @pemsley 
I added io.github.pemsley.coot.svg, which will be used as Flatpak app icon.

**Comment by eunos-1128** [comment]
Can you check the files and if everything is ok, can you please merge them?

**Pull request merged**